### ISSUE TITLE
Partners: When adding a New Partner that is Marked for Deletion in VISION, this flag (or indicator) does not get carried over. ALM 188

### ISSUE THREAD

**Issue by mpawlowski85** [unicef/etools-issues#689]
Browser: Chrome
Workspace: Jordan

1. Click Add New Partner button.
2. Enter Vendor No: 2500213083

    This Vendor is Marked for Deletion in VISION.
Result: Marked for deletion flag is not carried over

Another way:
1. Open a Partner that has Status =  "Marked for Deletion in VISION". Ex Vendor No 2500205900.
2. Delete this Partner.
3. Add this Partner back and Save.
4. Now this Partner has lost the Status "Marked for Deletion in VISION".

**Comment by ntrncic** [comment]
@mpawlowski85 yes we have missed this in our code, but why would you add a partner that has been marked for deletion in VISION?

**Comment by mpawlowski85** [comment]
@ntrncic it was just a way to test if the "marked for deletion" marker gets passed on from VISION - you would not add anyone who is marked for deletion in a normal course of business, but a partner might become marked for deletion at a later stage. 
Also - please do not forget about the "blocked" marker - this also does not seem to show up.
